sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/models/_patch.py

```diff
@@ -1,92 +1,217 @@
-# ------------------------------------
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT License.
-# ------------------------------------
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------
 """Customize generated code here.
 
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
-from typing import List, Optional, Tuple, Union, cast, Any
-from ._models import (
-    MetadataFilter as MetadataFilterGenerated,
-    AnswersFromTextOptions as AnswersFromTextOptionsGenerated,
-    TextDocument,
-    JSON,
-)
-
-
-class MetadataFilter(MetadataFilterGenerated):
-    """Find QnAs that are associated with the given list of metadata.
-
-    :ivar metadata:
-    :vartype metadata: list[tuple[str, str]]
-    :ivar logical_operation: Operation used to join metadata filters. Possible values include:
-     "AND", "OR".
-    :vartype logical_operation: str
+
+from __future__ import annotations
+
+from collections.abc import Iterable, Mapping, MutableMapping
+from typing import Any, Optional, Sequence, Tuple, Union, Mapping as TypingMapping, List, overload
```

Comment:
You don't need separate imports of Mapping from typing vs collections.abc - they are aliases for the same object:
https://docs.python.org/3/library/typing.html#typing.Mapping

So you only need to import once from collections.abc